TeslaSUI/TeslaSUI/Views/CustomTabView/CustomTabView.swift

```diff
@@ -0,0 +1,82 @@
+// CustomTabView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Кастомный табвью
+struct CustomTabView<Content: View>: View {
+    // MARK: - Public Properties
+
+    @Binding var selection: Int
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            content
+            CustomTabViewBackground()
+                .fill(Color.black)
+                .shadow(color: .gray, radius: 3)
+                .ignoresSafeArea()
+            HStack {
+                tabsView
+            }
+        }
+        .onPreferenceChange(TabItemPreferenceKey.self) { value in
+            customTabViewModel.tabs = value
+        }
+    }
+
+    // MARK: - Init
+
+    public init(selection: Binding<Int>, @ViewBuilder content: () -> Content) {
+        self.content = content()
+        _selection = selection
+    }
```

Comment:
```suggestion
    // MARK: - Init

    public init(selection: Binding<Int>, @ViewBuilder content: () -> Content) {
        self.content = content()
        _selection = selection
    }
```
инит после приватных проперти